src/planner/operator/logical_cte.cpp

```diff
@@ -0,0 +1,19 @@
+#include "duckdb/planner/operator/logical_cte.hpp"
+#include "duckdb/common/field_writer.hpp"
+
+namespace duckdb {
+
+void LogicalCTE::Serialize(FieldWriter &writer) const {
```

Comment:
The `LogicalCTE` is only created for materialized CTEs, right? Could we perhaps rename it to `LogicalMaterializedCTE` for more clarity?

Reply:
Makes sense. It is renamed now.